src/uwtools/drivers/ungrib.py

```diff
@@ -29,23 +29,22 @@ def __init__(
         """
         The driver.
 
-        :param cycle: The forecast cycle.
+        :param cycle: The synoptic time.
         :param config: Path to config file (read stdin if missing or None).
         :param dry_run: Run in dry-run mode?
         :param batch: Run component via the batch system?
         """
-        super().__init__(config=config, dry_run=dry_run, batch=batch)
-        self._config.dereference(context={"cycle": cycle})
+        super().__init__(config=config, dry_run=dry_run, batch=batch, cycle=cycle)
         if self._dry_run:
             dryrun()
         self._cycle = cycle
 
     # Workflow tasks
 
     @task
-    def gribfile_aaa(self):
+    def gribfile(self):
```

Comment:
Renamed to be less specific, as the `.AAA` suffix on the symlink is a detail that probably only matters to `ungrib` and not something users need to know about. Plus, we may later call this task to create additional symlinks with extensions like `.AAB`, etc., when configuring `ungrib` to process multiple files at once.